ocaml/xapi/helpers.ml

```diff
@@ -1312,3 +1312,7 @@ let retry ~__context ~doc ?(policy = Policy.standard) f =
 
 let retry_with_global_lock ~__context ~doc ?policy f =
   retry ~__context ~doc ?policy (fun () -> with_global_lock f)
+
+let vdi_operation_can_be_performed_live op = match op with
+  | `snapshot | `resize_online | `blocked | `clone | `mirror | `copy -> true
```

Comment:
Originally I place this function in `xapi_vdi_helpers.ml` and invoke it in `xapi_vbd_helpers.ml`, but unfortunately, it causes circular reference, so I move the function to `helpers.ml`. @jonludlam Could you suggest a better place store this function?